### PR TITLE
Support call to built-in `tuple()`

### PR DESCRIPTION
**Pull request by imperial-chief**
Solve #1571: add support for the built-in Python function `tuple` (which only takes one argument and unpacks any variables). Modifications:

- Add the class `PythonTupleFunction` representing a call to the `tuple` function.
- Add multiple test cases.

### CODE REVIEW

**Comment by EmilyBourne** [comment]
/bot run docs spelling pylint pyccel_lint coverage

**Comment by EmilyBourne** [comment]
/bot run docs spelling pylint pyccel_lint coverage

**Comment by EmilyBourne** [comment]
/bot run docs spelling pylint pyccel_lint coverage

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ All notable changes to this project will be documented in this file.
 
 -   #1587 : Fix unnecessarily long file names generated by `epyccel`.
 
+### Added
+
+-   #1571 : Add support for the function `tuple`.
```
Comment (EmilyBourne) [inline CHANGELOG.md:14]: There is already an "Added" section above. Please can you move the bullet point to the existing section

**Review by EmilyBourne — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,13 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+-   #1571 : Add support for the function `tuple`.
+
 ### Fixed
 
 -   #1587 : Fix unnecessarily long file names generated by `epyccel`.
 
+
```
Comment (yguclu) [inline CHANGELOG.md:14]: Please remove useless empty line.

**Review by yguclu — CHANGES_REQUESTED** [review]
Looks good to me!

File: CHANGELOG.md (hunk 2 above)
Comment (yguclu) [reply CHANGELOG.md]: Done in f0941c2.

**Review by yguclu — APPROVED** [review]

**Comment by yguclu** [comment]
\bot run pr_tests

**Pull request merged**